examples/2d/cifar_resnet_torch.py

```diff
@@ -158,20 +158,18 @@ def test(model, device, test_loader, scattering):
     else:
         scattering = Scattering2D(J=2, shape=(32, 32))
         K = 81*3
-    if use_cuda:
-        scattering = scattering.cuda()
+    scattering = scattering.to(device)
 
 
 
 
     model = Scattering2dResNet(K, args.width).to(device)
 
     # DataLoaders
+    num_workers = 4
     if use_cuda:
-        num_workers = 4
         pin_memory = True
     else:
-        num_workers = None
```

Comment:
?

Reply:
Why should number of cpu workers change depending on if we're on cpu or not?

Reply:
Agreed, but this was maybe there for a reason. Does it affect performance?

Reply:
`num_workers = None` appears to give me an [error](https://colab.research.google.com/drive/10_bgL7GRPvrpLiMBxd__aK1mtdj3U4wr#scrollTo=0Pp33Tr7qZIM). 

I don't think there are many reasons to set `num_works = 0 ` when most modernday cpus are multicore. @eugenium care to chime in?

Reply:
Ok.